docs/pages/access-controls/guides/device-trust.mdx

```diff
@@ -269,14 +302,38 @@ ERROR: ssh: rejected: administratively prohibited (unauthorized device)
 ```
 
 <Admonition type="tip" title="Trusted Clusters">
-It is possible to use [trusted
-clusters](../../management/admin/trustedclusters.mdx) to limit the impact of
-device mode `required`. A leaf cluster in mode `required` will enforce access to
-all of its resources, without imposing the same restrictions to the root
-cluster. Likewise, a root cluster will not enforce device trust on resources in
-leaf clusters.
+  It is possible to use [trusted
+  clusters](../../management/admin/trustedclusters.mdx) to limit the impact of
+  device mode `required`. A leaf cluster in mode `required` will enforce access
+  to all of its resources, without imposing the same restrictions to the root
+  cluster. Likewise, a root cluster will not enforce device trust on resources
+  in leaf clusters.
 </Admonition>
 
+## Locking device
+
+Similar to [session and identity locking](./locking.mdx), a device can be locked using `tctl lock`.
```

Comment:
We should explain here what locking a device means and how it takes effect. Teleport doesn't identify trusted devices, so if device trust is disabled or a device is unenrolled, locking it will have no effect. This can be confusing for users, so let's spell  it out.

There are some details on https://github.com/gravitational/teleport/pull/21667 too.

Reply:
Please update `locking.mdx` too  (list a device as a valid lock target)

Reply:
Resolved in following commit. Added device lock target in `locking.mdx` as well.